src/basic/process-util.c

```diff
@@ -2060,47 +2056,41 @@ int posix_spawn_wrapper(
                 return -r;
 
 #if HAVE_PIDFD_SPAWN
-        if (clone_support >= CLONE_CAN_PIDFD) {
```

Comment:
as above: i think we might to keep this a bit longer, for compat with outdated seccomp filter sets in container managers.